pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -217,3 +230,66 @@ func InjectSpanID(executionContext *ExecutionStartInfo, headers http.Header) {
 		executionContext.SpanID = value
 	}
 }
+
+func capturePayloadAsTags(value interface{}, targetSpan *pb.Span, key string, depth int, maxDepth int) {
+	if key == "" {
+		return
+	}
+	if value == nil {
+		targetSpan.Meta[key] = ""
+		return
+	}
+	if depth >= maxDepth {
```

Comment:
Am I groking this wrong or will this case never get hit? depth is 0 and maxDepth would be a positive int right? so 0 >= positive will never hit?

Reply:
look for the (depth +1) call or look for the test case.